contrib/storage-jdbc/src/test/resources/mysql-test-data.sql

```diff
@@ -0,0 +1,53 @@
+
+set global time_zone = "+00:00";
+
+use drill_mysql_test;
+
+create table person (
+  person_id       INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+
+  first_name      VARCHAR(255),
+  last_name       VARCHAR(255),
+  address         VARCHAR(255),
+  city            VARCHAR(255),
+  state           CHAR(2),
+  zip             INT,
+
+  json            VARCHAR(255),
+
+  bigint_field    BIGINT,
+  smallint_field  SMALLINT,
+  numeric_field   NUMERIC(10, 2),
+  boolean_field   BOOLEAN,
+  double_field    DOUBLE,
+  float_field     FLOAT,
+  real_field      REAL,
+
+  time_field      TIME,
+  timestamp_field TIMESTAMP,
+  date_field      DATE,
+  datetime_field  DATETIME
```

Comment:
If I recall correctly, we previously had issues with TEXT, BLOB, CLOB and BIT columns. Can you add these column types to the tests?